pkgs/development/python-modules/pslab/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, numpy, pyserial, scipy
+, pytest-mock, pytest, coverage}:
+
+buildPythonPackage rec {
+  pname = "pslab";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "fossasia";
+    repo = "pslab-python";
+    rev = "v" + version;
+    sha256 = "12n2zdbgbhrnkgf1z7hlpvgl8zycvw3qxc6fhh1srkj6j2jwvr2n";
+  };
+
+  propagatedBuildInputs = [ numpy pyserial scipy ];
+  checkInputs = [ pytest pytest-mock coverage ];
```

```suggestion
  checkInputs = [ pytestCheckHook pytest-mock ];
```
We don't want to generate coverage.